curvefs/src/client/fuse_s3_client.cpp

```diff
@@ -246,22 +266,36 @@ void FuseS3Client::travelChunk(fuse_ino_t ino, S3ChunkInfoList chunkInfo,
               blockIndex <= travelEndIndex ; blockIndex++) {
                 auto objectName = curvefs::common::s3util::GenObjName(
                     chunkid, blockIndex, compaction, fsId, ino);
-                prefetchObjs->push_back(std::make_pair(objectName, blockSize));
+                {
+                    std::unique_lock<std::mutex> lck(warmupObjsMtx_);
+                    needWarmupObjs_.push_back(
+                      std::make_pair(objectName, blockSize));
+                }
             }
         }
     }
 }
 
 // TODO(hzwuhongsong): These logics are very similar to other place,
 // try to merge it
-void FuseS3Client::WarmUpAllObjs(
-  const std::list<std::pair<std::string, uint64_t>> &prefetchObjs) {
+void FuseS3Client::WarmUpAllObjs() {
+    std::list<std::pair<std::string, uint64_t>> needWarmupObjs;
+    {
+        std::unique_lock<std::mutex> lck(warmupObjsMtx_);
+        LOG(INFO) << "num of objs need loaded is: " << needWarmupObjs_.size();
```

Comment:
How many times will this log be printed in a warmup?

Reply:
maybe several times